### DISCUSSION TITLE
Translating difficulties:

### DISCUSSION THREAD

**Discussion by skanda890** [Devolutions/UniGetUI#2829] [General]
Some phrases that have a colon are difficult to translate in Sanskrit because some words have something like the colon at the end, and what should I do if there is something like this {0} or like { } in the middle of the phrases?

**Comment by marticliment** [comment]
What do you mean by _something like the colon_?

**Reply by marticliment** [reply]
Yes, I know what a colon is.
I need you to provide more details, and examples on which keys can't be accurately translated

**Reply by skanda890** [reply]
The phrase Application theme: with a colon is translated into Sanskrit is अनुप्रयोगस्य विषयः: it has two colon-like things. Also, the phrase Architecture to install: with a colon is translated into Sanskrit is स्थापयितुं वास्तुशिल्पः: it has two colon-like things.

**Reply by marticliment** [reply]
So what do you propose to fix this?

**Reply by skanda890** [reply]
Rephrase those sentences in Tolgee and UniGetUI.

**Reply by skanda890** [reply]
Do you have any suggestions?

**Reply by marticliment** [reply]
You are free to rephrase them on Sanskrit as long as they make sense on the interface.

**Reply by skanda890** [reply]
But I couldn't find synonyms for those words. What should I do??

**Reply by marticliment** [reply]
Then just leave them as they are, with the double colons

**Reply by skanda890** [reply]
Then what about the phrases with {} in the middle?

**Reply by marticliment** [reply]
Those {i} must be maintained intact, because they will be replaced at some point for some content. 
For example:
 - Are you sure you want to remove {0} from your computer?
translates to
 - Are you sure you want to remove Google Chrome from your computer?

**Reply by skanda890** [reply]
I have a problem logging into Tolgee Cli. Please tell me the command you use to log into Tolgee Cli. Make sure you replace your API key with "your-API-key". 
Issue = tolgee/tolgee-cli#118

**Reply by marticliment** [reply]
Please test if this issue still happens with Tolgee CLI 1.4.0 or Tolgee CLI 2.0.0

**Reply by skanda890** [reply]
How can I download Tolgee CLI 2.2.0 or 2.0.0?

**Reply by marticliment** [reply]
From npm: npm install --global @tolgee/cli@your_version

**Reply by skanda890** [reply]
> npm install --global @tolgee/cli@your_version

Still doesn't work.

**Reply by marticliment** [reply]
I am afraid I can't help with that, as I don't use tolgee-cli. I use the web-interface
